service/test/controllers/usermanagement/UserControllerTest.java

```diff
@@ -1,95 +1,58 @@
 import java.util.Map;
-import org.junit.BeforeClass;
-import org.junit.FixMethodOrder;
-import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.MethodSorters;
-import org.mockito.Mockito;
-import org.powermock.api.mockito.PowerMockito;
-import org.powermock.core.classloader.annotations.PowerMockIgnore;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
 import org.sunbird.common.models.util.JsonKey;
 import org.sunbird.common.models.util.ProjectLogger;
-import org.sunbird.common.request.HeaderParam;
 import play.libs.Json;
 import play.mvc.Http.RequestBuilder;
 import play.mvc.Result;
-import play.test.FakeApplication;
-import play.test.Helpers;
-import util.RequestInterceptor;
-
-/** Created by arvind on 6/12/17. */
-@FixMethodOrder(MethodSorters.NAME_ASCENDING)
-@RunWith(PowerMockRunner.class)
-@PrepareForTest(RequestInterceptor.class)
-@PowerMockIgnore("javax.management.*")
-@Ignore
-public class UserControllerTest {
-
-  private static FakeApplication app;
-  private static Map<String, String[]> headerMap;
-  private static ActorSystem system;
-  private static final Props props = Props.create(DummyActor.class);
-
-  @BeforeClass
-  public static void startApp() {
-    app = Helpers.fakeApplication();
-    Helpers.start(app);
-    headerMap = new HashMap<String, String[]>();
-    headerMap.put(HeaderParam.X_Consumer_ID.getName(), new String[] {"Service test consumer"});
-    headerMap.put(HeaderParam.X_Device_ID.getName(), new String[] {"Some Device Id"});
-    headerMap.put(
-        HeaderParam.X_Authenticated_Userid.getName(), new String[] {"Authenticated user id"});
-    headerMap.put(JsonKey.MESSAGE_ID, new String[] {"Unique Message id"});
-
-    system = ActorSystem.create("system");
-    ActorRef subject = system.actorOf(props);
-    BaseController.setActorRef(subject);
-  }
+
+public class UserControllerTest extends BaseControllerTest {
+
+  private static String userId = "2a8316e3-7da4-4001-8494-90f40b1853e8";
+  private static String emailId = "abbc@gmail.com";
+  private static String phoneNumber = "8800088000";
+  private static String userName = "userName";
+  private static String orgId = "0126180523645747200";
+  private static String loginId = "loginid";
+  private static String invalidPhonenumber = "00088000";
+  private static String firstName = "firstName";
+  private static String lastName = "lastName";
+  private static String query = "query";
+  private static String language = "hindi";
+  private static String role = "user";
 
   @Test
-  public void testCreateUserWithoutContentType() {
-    PowerMockito.mockStatic(RequestInterceptor.class);
-    when(RequestInterceptor.verifyRequestData(Mockito.anyObject()))
-        .thenReturn("{userId} uuiuhcf784508 8y8c79-fhh");
+  public void testCreateUserFailureWithoutContentType() {
     Map<String, Object> requestMap = new HashMap<>();
```

Comment:
Complete request object can be created  in a separate method.  Avoid duplicate code.